pkg/apis/yugabyte/v1alpha1/ybcluster_types.go

```diff
@@ -18,6 +18,7 @@ type YBClusterSpec struct {
 	// Add custom validation using kubebuilder tags: https://book-v1.book.kubebuilder.io/beyond_basics/generating_crd.html
 	// +kubebuilder:validation:Minimum=1
 	ReplicationFactor int32         `json:"replicationFactor,omitempty"`
+	Domain            string        `json:"domain,omitempty"`
```

Comment:
```suggestion
	// Domain name for Kubernetes cluster
	Domain            string        `json:"domain,omitempty"`
```

Run `operator-sdk generate k8s` and `operator-sdk generate crds`, and then `go fmt` after changing this.